source/install-riallto-linux.rst

```diff
@@ -0,0 +1,59 @@
+.. _install-riallto-Linux:
+
+Install Riallto on Linux 
+===========================
+
+Currently there is support for bringing up Riallto on Ubuntu 24.04 with docker.
+To use Riallto on Linux requires the use of the [xdna-driver](https://github.com/amd/xdna-driver) which is installed as part of the setup.
+This driver requires version 6.10+ of the linux kernel, these scripts will upgrade a standard Ubuntu 24.04 installation to this kernel version. __Using this kernel version will require disabling secure boot on your device.__
```

Comment:
Is it confirmed that secure boot needs to be disabled?

Reply:
So it looks like if we waited for the 6.10 signed version to come out we wouldn't. However, it looks like the signed version comes out a month after the release of the kernel, so I think that would be too long a wait to push back v1.1.

Reply:
What's the plan, update the docs once the signed version is there?

Reply:
That's an option I guess, there is nothing on open downloads etc... that would need to change.